### ISSUE TITLE
Panel icon for indicator-brightness

### ISSUE THREAD

**Issue by mgajanssen** [numixproject/numix-icon-theme#1153]
Indicator-brightness allows you to adjust the screen brightness from the panel. More info here:
https://launchpad.net/indicator-brightness

http://bazaar.launchpad.net/~indicator-brightness/indicator-brightness/trunk/view/head:/notification-display-brightness-full.svg

**Comment by wa4557** [comment]
I suppose this is a wontfix, because it would interfere with the actual notification icons that are already present in numix

**Comment by Foggalong** [comment]
Ooo yeah it will. What happens if we were to add that icon at panel sizes?

**Comment by wa4557** [comment]
On my machine the notification icon (when changing the brightness) looks blurry then :/

**Comment by palob** [comment]
Are notification icons shown at 48px? If yes, I think there must be a 48px icon (along other sizes) or a 16px icon only in order for it not to be blurry.

**Comment by wa4557** [comment]
Yes they are shown 48px on unity whenever you change screen brightness. For me this is a wontfix, this is not a status but a notification icon

**Comment by palob** [comment]
@martijnjanssen So is this shown in the panel? Can you please take screenshots of where the original icon pops up?

**Comment by martijnjanssen** [comment]
Unfortunaltely I'm not using indicator brightness anymore due to gnome having another solution for this, but the icon is solely used in the top bar next to the wifi icon. This is a way of controlling the brightness from the top bar without using the media keys.
